src/browser/setupWorker/glossary.ts

```diff
@@ -75,24 +75,6 @@ export interface StringifiedResponse extends ResponseInit {
   body: string | ArrayBuffer | ReadableStream<Uint8Array> | null
 }
 
-/**
- * Map of the events that can be sent to the Service Worker
- * only as a part of a single `fetch` event handler.
- */
-export interface ServiceWorkerFetchEventMap {
```

Comment:
This is a leftover from the previous client/worker communication protocol. We moved these types to the event channel now (found by looking at `createRequestListener`).